src/_includes/globals/service-worker.njk

```diff
@@ -1,6 +0,0 @@
-<!-- Add service worker -->
-<script>
-  if ("serviceWorker" in navigator) {
-    navigator.serviceWorker.register("/service-worker.js");
-  }
-</script>
```

Comment:
@cindyli Was your intention to remove the service worker? Now it is being built properly (and appears in the build directory as `service-worker.js`) but by removing this file and the include in `layouts/base.njk` it is no longer being loaded. This will pose a problem as the old service worker will still be registered for visitors who visited before it broke, which means they may still see out of date content. If you want to remove the service worker altogether you would need to follow the steps here: https://github.com/lwojcik/eleventy-plugin-pwa-v2#uninstalling-the-plugin

Reply:
I was intended to upgrade the service worker but didn't understand `service-worker.js` was generated by the updated package in the build directory. :) Thanks for pointing out, @greatislander. It has been fixed and ready for another look.